src/openforms/js/components/admin/form_design/variables/ServiceFetchConfigurationPicker.js

```diff
@@ -114,8 +125,10 @@ const ServiceFetchConfigurationPicker = ({onChange, onFormSave}) => {
 };
 
 ServiceFetchConfigurationPicker.propTypes = {
+  variableName: PropTypes.string.isRequired,
   onChange: PropTypes.func.isRequired,
   onFormSave: PropTypes.func.isRequired,
+  initialValues: PropTypes.object,
```

Comment:
```suggestion
  initialValues: PropTypes.shape({...}),
```
we know ahead of time what the shape of this object is, right? So we should be able to make a strict and explicit proptype